blueprints/app/files/.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   root: true,
   parser: '<%= typescript ? '@typescript-eslint/parser' : '@babel/eslint-parser' %>',
   parserOptions: {
-    ecmaVersion: 2018,<% if (!typescript) { %>
+    ecmaVersion: 2022,<% if (!typescript) { %>
```

Comment:
Can we use `'latest'` instead? I think that's the default as well in the new ESLint config system.